scripts/solr_builder/solr_builder/solr_builder.py

```diff
@@ -107,7 +105,13 @@ def query_iter(self, query, size=20):
 
         cur.close()
 
-    def query_batched(self, query, size, cursor_name=None, cache_json=False):
+    def query_batched(
+        self,
+        query: str,
+        size: int,
+        cursor_name: Optional[str] = None,
```

Comment:
```suggestion
        cursor_name: str = None,
```
For function and method parameters with a default value of `None` we can remove the `Optional[]` because both Python and mypy are smart enough to figure this out.  The hint is telling the caller to provide a `str` for this parameter but it is OK it the parameter is not provided.  Doing this greatly decreases the visual clutter.